example/src/tests/groupTests.ts

```diff
@@ -800,3 +803,60 @@ test('can paginate group messages', async () => {
 
   return true
 })
+
+test('can stream all group messages', async () => {
+  const bo = await Client.createRandom({ env: 'local', enableAlphaMls: true })
+  await delayToPropogate()
+  const alix = await Client.createRandom({ env: 'local', enableAlphaMls: true })
+  await delayToPropogate()
+
+  // Start Bob starts a new group.
+  const boGroup = await bo.conversations.newGroup([alix.address])
+  await delayToPropogate()
+
+  // Starts a new conversation.
+  const caro = await Client.createRandom({ env: 'local', enableAlphaMls: true })
+  const caroGroup = await caro.conversations.newGroup([alix.address])
+
+  // Record message stream across all conversations
+  const allMessages: DecodedMessage[] = []
+  await alix.conversations.streamAllGroupMessages(async (message) => {
+    allMessages.push(message)
+  })
+
+  for (let i = 0; i < 5; i++) {
+    await boGroup.send({ text: `Message ${i}` })
+    await delayToPropogate()
+  }
+
+  const count = allMessages.length
+  if (count !== 5) {
+    throw Error('Unexpected all messages count first' + allMessages.length)
+  }
+
+  await delayToPropogate()
+  for (let i = 0; i < 5; i++) {
+    await caroGroup.send({ text: `Message ${i}` })
+    await delayToPropogate()
+  }
+
+  if (allMessages.length !== 10) {
+    throw Error('Unexpected all messages count second' + allMessages.length)
+  }
+
+  alix.conversations.cancelStreamAllGroupMessages()
+
```

Comment:
Figured out that this test only passes for me on Android if I have an additional delay between the`cancelStreamAllGroupMessages()` and starting the stream again. Otherwise `allMessages.length` would equal 10 for me in the last check. 

```suggestion
  await delayToPropogate()
```